File: scratch/validate_json.js
const fs = require('fs');

try {
    const buffer = fs.readFileSync('c:\\Users\\Yacine\\Downloads\\Mission_R1_Complet.json');
    console.log("First 10 bytes (hex):", buffer.slice(0, 10).toString('hex'));
    const data = buffer.toString('utf8');
    JSON.parse(data);
    console.log("JSON is valid");
} catch (e) {
    console.log("JSON is invalid: " + e.message);
}

File: scratch/validate_json.py
import json
import sys

try:
    with open(r'c:\Users\Yacine\Downloads\Mission_R1_Complet.json', 'r', encoding='utf-8') as f:
        data = json.load(f)
    print("JSON is valid")
except Exception as e:
    print(f"JSON is invalid: {e}")
